### PR TITLE
mmrcms split

### LINKED ISSUE #6466: [Meta] Split mmrcms to multisrc

### PR DESCRIPTION
**Pull request by jopejoe1**
To-Do:

- [x] Implement Old generator / or some other way to make it easier to add sources
- [x] add icons
- [x] test all sources
- [x] change stuff to work with #6742
- [x] Fix bug
- [-] ~~Improve Hentai Shark language selection~~ Not doing it for now maybe later

Closes #6466

### CODE REVIEW

**Comment by jopejoe1** [comment]
Removed:
- **Mangas Yuri** (Sells Manga now)
- **frscan** (Is offline from what I see)

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/MMRCMSGenerator.kt [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+class MMRCMSGenerator : ThemeSourceGenerator {
+
+    override val themePkg = "mmrcms"
+
+    override val themeClass = "MMRCMS"
+
+    override val baseVersionCode: Int = 1
+
+    override val sources = listOf(
+        SingleLang("مانجا اون لاين", "https://onma.me", "ar", className = "onma"),
```
Comment (AriaMoradi) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/MMRCMSGenerator.kt:15]: Capitalize class name?

**Review by AriaMoradi — CHANGES_REQUESTED** [review]

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/MMRCMSGenerator.kt (hunk 1 above)
Comment (jopejoe1) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mmrcms/MMRCMSGenerator.kt]: Yeah, seems right didn't think about that.

**Comment by papjul** [comment]
> Removed:
> 
>     * **[frscan](https://www.frscan.me/)** (Is offline from what I see)

I don't know what the old website looked like, but I believe it's frscan.cc now?

**Comment by jopejoe1** [comment]
> > Removed:
> > ```
> > * **[frscan](https://www.frscan.me/)** (Is offline from what I see)
> > ```
> 
> I don't know what the old website looked like, but I believe it's frscan.cc now?

Added frscan.cc

**Comment by jopejoe1** [comment]
Should be ready for merge

**Comment by CarlosEsco** [comment]
Unmark it draft?  How does this source work now?  Run MMRCMSJsonGen and thats it?

**Comment by jopejoe1** [comment]
Found a major bug that i need to fix first thats why i put it back into draft.

**Comment by jopejoe1** [comment]
Also need to change stuff to work with #6742

**Comment by jopejoe1** [comment]
> Unmark it draft?

See comments above

> How does this source work now? Run MMRCMSJsonGen and thats it?

That was the intended way, not sure if its still gone be like that after i fixe the bug.

**Pull request closed without merging**